src/main/java/com/whereisagift/wishlist/WishlistService.java

```diff
@@ -0,0 +1,71 @@
+package com.whereisagift.wishlist;
+
+import com.whereisagift.user.UserRepository;
+import com.whereisagift.wish.Wish;
+import com.whereisagift.wish.WishRepository;
+import graphql.GraphQLException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.nio.file.AccessDeniedException;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class WishlistService {
```

Comment:
```suggestion
@Service
@Transactional
@RequiredArgsConstructor
public class WishlistService {
```

Тогда не надо над каждой ручкой вешать эту аннотацию